File: app/models/ticket.rb
class Ticket < ActiveRecord::Base
  	belongs_to :route
	belongs_to :user
	belongs_to :stop_start, :class_name => "Stop"
	belongs_to :stop_end, :class_name => "Stop"
  attr_accessible :date, :paid, :price, :route_id, :user_id, :stop_start_id, :stop_end_id
end

File: Android/src/tickets/Ticket.java
package tickets;

public class Ticket {
	public String date;
	public String from;
	public String to;
	public double price;
	public boolean paid;

	public Ticket() {
		super();
	}

	public Ticket(String date, String from, String to, double price, boolean paid) {
		super();
		this.date = date;
		this.from = from;
		this.to = to;
		this.price = price;
		this.paid = paid;
	}
}
